packages/docusaurus-theme-translations/locales/pl/theme-search-algolia.json

```diff
@@ -1,29 +1,29 @@
 {
   "theme.SearchBar.label": "Szukaj",
   "theme.SearchBar.seeAll": "Wyświetl wszystkie {count} wyników",
-  "theme.SearchModal.errorScreen.helpText": "You might want to check your network connection.",
-  "theme.SearchModal.errorScreen.titleText": "Unable to fetch results",
-  "theme.SearchModal.footer.closeKeyAriaLabel": "Escape key",
-  "theme.SearchModal.footer.closeText": "to close",
-  "theme.SearchModal.footer.navigateDownKeyAriaLabel": "Arrow down",
-  "theme.SearchModal.footer.navigateText": "to navigate",
-  "theme.SearchModal.footer.navigateUpKeyAriaLabel": "Arrow up",
-  "theme.SearchModal.footer.searchByText": "Search by",
-  "theme.SearchModal.footer.selectKeyAriaLabel": "Enter key",
-  "theme.SearchModal.footer.selectText": "to select",
-  "theme.SearchModal.noResultsScreen.noResultsText": "No results for",
-  "theme.SearchModal.noResultsScreen.reportMissingResultsLinkText": "Let us know.",
-  "theme.SearchModal.noResultsScreen.reportMissingResultsText": "Believe this query should return results?",
-  "theme.SearchModal.noResultsScreen.suggestedQueryText": "Try searching for",
-  "theme.SearchModal.placeholder": "Search docs",
-  "theme.SearchModal.searchBox.cancelButtonText": "Cancel",
-  "theme.SearchModal.searchBox.resetButtonTitle": "Clear the query",
-  "theme.SearchModal.startScreen.favoriteSearchesTitle": "Favorite",
-  "theme.SearchModal.startScreen.noRecentSearchesText": "No recent searches",
-  "theme.SearchModal.startScreen.recentSearchesTitle": "Recent",
-  "theme.SearchModal.startScreen.removeFavoriteSearchButtonTitle": "Remove this search from favorites",
-  "theme.SearchModal.startScreen.removeRecentSearchButtonTitle": "Remove this search from history",
-  "theme.SearchModal.startScreen.saveRecentSearchButtonTitle": "Save this search",
+  "theme.SearchModal.errorScreen.helpText": "Może warto sprawdzić połączenie sieciowe.",
+  "theme.SearchModal.errorScreen.titleText": "Nie można pobrać wyników",
+  "theme.SearchModal.footer.closeKeyAriaLabel": "Klawisz Escape",
+  "theme.SearchModal.footer.closeText": "do zamknięcia",
+  "theme.SearchModal.footer.navigateDownKeyAriaLabel": "Strzałka w dół",
+  "theme.SearchModal.footer.navigateText": "poruszać się",
```

Comment:
```suggestion
  "theme.SearchModal.footer.closeKeyAriaLabel": "Klawisz Escape",
  "theme.SearchModal.footer.closeText": "aby zamknąć",
  "theme.SearchModal.footer.navigateDownKeyAriaLabel": "Strzałka w dół",
  "theme.SearchModal.footer.navigateText": "aby przejść",
```

Those labels are used in search legend, describing the functionality of buttons.

<img width="251" alt="Screenshot 2023-01-17 100253" src="https://user-images.githubusercontent.com/719641/212854797-0c30be5c-aa63-4e1a-821f-1e8310bc56ba.png">